packages/storybook/stories/components/text/text.stories.ts

```diff
@@ -1,34 +1,47 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
 import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-text';
 import { ODS_TEXT_PRESETS } from '@ovhcloud/ods-components';
 import { html } from 'lit-html';
 import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
-import type { Meta } from '@storybook/web-components';
 
 defineCustomElement();
 
 const meta: Meta = {
-  title: 'ODS Components/Content/Text/Demo',
+  title: 'ODS Components/Content/Text',
+  component: 'ods-text',
+};
+
+export default meta;
+
+export const Demo: StoryObj = {
+  /*
+  * todo check why render is necessary here? https://storybook.js.org/docs/api/csf#default-render-functions
+  *  and it affects canvas rendering in documentation (presets are overrided for example)
+  *  */
   render: (args) => html`
     <ods-text preset="${args.preset}">
       ${unsafeHTML(args.content)}
     </ods-text>
   `,
   argTypes: {
     preset: {
+      table: {
+        category: 'General',
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' }
+      },
       control: { type: 'select' },
       options: ODS_TEXT_PRESETS,
     },
     content: {
+      table: {
+        category: 'General',
```

Comment:
would be nice to use the latest changes we discussed on controls: https://github.com/ovh/design-system/blob/refactor/input-attrs/packages/storybook/stories/controls.ts